ecr-login/go.mod

```diff
@@ -13,4 +13,20 @@ require (
 	github.com/stretchr/testify v1.8.2
 )
 
```

Comment:
looks like it is missing `go 1.19` to set the min supported go version?

Reply:
ahh, it is in L32. I usually expect the go directive to be present after `module`. can you move that here for clarity?